### DISCUSSION TITLE
show created menu on mobile

### DISCUSSION THREAD

**Discussion by imed-ghomari** [Taitava/obsidian-shellcommands#212] [Ideas - on hold]
When holding over a link in mobile (right click menu on desktop), I don't see the custom menus created by the plugin. Is this the intended behavior ?

**Comment by Taitava** [comment]
Unfortunately mobile platforms are not supported, because I don't know if it's possible to execute shell commands on mobile by using JavaScript libraries Obsidian uses on mobile. At least `Node.js` libraries that are used on the desktop version, are not available on mobile AFAIK. I'd like to know if there's an alternative way to execute shell commands on mobile.

Normally this plugin should not even be able to install on mobile devices. Are you using Obsidian Sync? I just noticed a vault of mine has the _Shell commands_ plugin installed on my iPhone, but I guess it's because I have used Obsidian Sync a few months ago. I don't know any other reason why the plugin is installed on my phone.

**Reply by imed-ghomari** [reply]
Ah I'm using syncthing, didn't know that It didn't work on mobile, thank you for the explanation.
